x/authz/migrations/v046/store.go

```diff
@@ -58,7 +62,10 @@ func addExpiredGrantsIndex(ctx sdk.Context, store storetypes.KVStore, cdc codec.
 		}
 	}
 
-	for key, v := range queueItems {
+	keys := maps.Keys(queueItems)
+	sort.Strings(keys)
+	for _, key := range keys {
```

Comment:
There is an absolute reason to, maps.Keys returns them in non-deterministic order and the source code shows it https://cs.opensource.google/go/x/exp/+/32f3d567:maps/maps.go;l=10
<img width="622" alt="Screen Shot 2022-10-28 at 9 16 39 PM" src="https://user-images.githubusercontent.com/4898263/198813423-a9d310b5-06d6-493d-ac93-38cf887471fa.png">

Reply:
Surprising! But if `map.Keys`  doesn't sort the keys, then the dependency provides very little value, right? It would be better to define your own `package map` with a `Keys` that returned ordered keys, I guess?